crates/breez-sdk/core/src/persist/sqlite.rs

```diff
@@ -62,7 +66,7 @@ impl SqliteStorage {
         };
         migrations.to_latest(&mut conn)?;
 
-        if previous_version < 6 {
+        if previous_version < 7 {
```

Comment:
Isn't this fine to stay as 6?

Reply:
Yes, I got confused about the numbers. This change was not necessary.